examples/dt/uni04delta/control-plane/service-values.yaml

```diff
@@ -0,0 +1,131 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  cinderBackup:
+    customServiceConfig: |
+      [DEFAULT]
+      backup_driver = cinder.backup.drivers.ceph.CephBackupDriver
+      backup_ceph_pool = backups
+      backup_ceph_user = openstack
+
+  cinderVolumes:
+    ceph:
+      customServiceConfig: |
+        [DEFAULT]
+        enabled_backends = ceph
+
+        [ceph]
+        volume_backend_name = ceph
+        volume_driver = cinder.volume.drivers.rbd.RBDDriver
+        rbd_ceph_conf = /etc/ceph/ceph.conf
+        rbd_user = openstack
+        rbd_pool = volumes
+        rbd_flatten_volume_from_snapshot = False
+        rbd_secret_uuid = _replaced_
+
+  glance:
+    customServiceConfig: |
+      [DEFAULT]
+      enabled_backends = default_backend:rbd
+
+      [glance_store]
+      default_backend = default_backend
+
+      [default_backend]
+      rbd_store_ceph_conf = /etc/ceph/ceph.conf
+      store_description = "RBD backend"
+      rbd_store_pool = images
+      rbd_store_user = openstack
+    glanceAPIs:
+      default:
+        replicas: 3
+
+  manila:
+    enabled: true
+    manilaShares:
+      share1:
+        customServiceConfig: |
+          [DEFAULT]
+          enabled_share_backends = cephfs
+          enabled_share_protocols = cephfs
+
+          [cephfs]
+          driver_handles_share_servers = False
+          share_backend_name = cephfs
+          share_driver = manila.share.drivers.cephfs.driver.CephFSDriver
+          cephfs_conf_path = /etc/ceph/ceph.conf
+          cephfs_cluster_name = ceph
+          cephfs_auth_id = openstack
+          cephfs_volume_mode = 0755
+          cephfs_protocol_helper_type = CEPHFS
+    manilaAPI:
+      customServiceConfig: |
+        [DEFAULT]
+        enabled_share_protocols=nfs,cephfs
+
+  neutron:
+    customServiceConfig: |
+      [DEFAULT]
+      vlan_transparent = true
+      service_plugins = qos,ovn-router,trunk,segments,log
+      agent_down_time = 600
+      router_distributed = true
+      router_scheduler_driver = neutron.scheduler.l3_agent_scheduler.ChanceScheduler
+      debug = true
+      [agent]
+      report_interval = 300
+      [database]
+      max_retries = -1
+      db_max_retries = -1
+      [keystone_authtoken]
+      region_name = regionOne
+      memcache_use_advanced_pool = True
+      auth_type = password
+      [nova]
+      region_name = regionOne
+      endpoint_type = internal
+      [oslo_messaging_notifications]
+      driver = noop
+      [oslo_middleware]
+      enable_proxy_headers_parsing = true
+      [oslo_policy]
+      policy_file = /etc/neutron/policy.yaml
+      [placement]
+      region_name = regionOne
+      [ovs]
+      igmp_snooping_enable = true
+      [ovn]
+      ovn_emit_need_to_frag = true
```

Comment:
can disable this for now as per https://github.com/openstack-k8s-operators/architecture/pull/228

Reply:
done